### PR TITLE
1199 1298 laos eoc historic and forecast weather dashboards

### PR DESCRIPTION
**Pull request by IgorNadj**
### Changes:

- Add historic and forecast dashboards for Laos EOC
- Add limits to dashboard reports
- Add lodash to database package

---

### Screenshots:
See https://github.com/beyondessential/tupaia-backlog/issues/1199#issuecomment-706813336

### CODE REVIEW

**Comment by IgorNadj** [comment]
Previous PR: https://github.com/beyondessential/tupaia/pull/1315

File: packages/database/package.json [hunk 1]
```diff
@@ -43,6 +43,7 @@
     "db-migrate-pg": "^1.2.2",
     "dotenv": "^8.2.0",
     "knex": "0.14.6",
+    "lodash": "^4.17.4",
```
Comment (rohan-bes) [inline packages/database/package.json:46]: I think we typically just add a dependency for the required function from `lodash`, eg. `"lodash.cloneDeep": ...`

**Review by rohan-bes — APPROVED** [review]
Looking real good 😄

Just one request to use the function when adding lodash dependency, but approving anyway 👍

File: packages/database/package.json (hunk 1 above)
Comment (IgorNadj) [reply packages/database/package.json]: Good point, updated

**Comment by IgorNadj** [comment]
Closing, branch name too long. Replaced by https://github.com/beyondessential/tupaia/pull/1581

**Pull request closed without merging**